cudamapper/samples/sample_cudamapper.cpp

```diff
@@ -29,6 +29,7 @@
 #include <iostream>
 #include <string>
 #include <mutex>
+#include <unistd.h>
```

Comment:
Why is this needed?

Reply:
I'm actually not sure - removed in 405ec7e4e0b090a3323da5970c038a219078c5bc.